pyproject.toml

```diff
@@ -35,15 +32,14 @@ keywords = [
     'ndx-extension',
 ]
 dependencies = [
-    "pynwb>=2.6.0",
-    "hdmf>=3.13.0",
+    "pynwb>=2.8.1",
+    "hdmf>=3.14.2",
 ]
```

Comment:
I can confirm this works with `pynwb==2.8.0` and `hdmf==3.14.1`
maybe we pin it to the lowest minor versions? @pauladkisson